01_Exercises/main.py

```diff
@@ -0,0 +1,74 @@
+# LAB 1
```

Comment:
Noteaza notele:

Nota 8 pentru sum_numbers.
Nota 9 pentru only_prime_numbers.
Nota 7 pentru highest_divisors.

Reply:
Happy with that.